app/controllers/users_controller.rb

```diff
@@ -10,7 +10,20 @@ def create
 
     if @user.save
       handle_invitation
-      AppMailer.delay.send_welcome_email(@user) 
+
+      Stripe.api_key = ENV['STRIPE_API_KEY']
+      begin
+        charge = Stripe::Charge.create(
+          :amount => 999,
```

Comment:
I would recommend to use Ruby 1.9 hash syntax, just to remain consistent. For instance, couple lines below [here]https://github.com/dcousette/myflix/pull/9/files#diff-4e05ad0d64e6100656b63ad1e78f32c5L25) you are using it and here not. It looks strange, when you mix it up.